lib/pltr/v2/selectors/characters.js

```diff
@@ -45,7 +45,12 @@ export const singleCharacterSelector = createSelector(
 )
 
 const displayedSingleCharacter = (character, bookId, currentBookAttributeDescirptorsById) => {
-  const currentBookAttributes = character.attributes || []
+  const currentBookAttributes =
+    character.attributes && Array.isArray(character.attributes)
+      ? character.attributes
+      : typeof character.attributes === 'string'
```

Comment:
@Quiescent i tested an old version of example3.pltr and exported it as scrivener and then import and then i got an error. It rendered an attribute as String

Reply:
@jeanawhou Hmmm.  That sounds like a bug in the importer.
It's not valid for `attributes` to be anything except for an array.
I think you'll need to investigate further and find out why the importer made it a string instead of an array.

Reply:
@Quiescent it actually work just now. I pulled the latest from the release branch, maybe that fixed it.. Im not sure :sweat_smile: